app/jobs/make_phone_calls_job.rb

```diff
@@ -0,0 +1,28 @@
+class MakePhoneCallsJob < ApplicationJob
+  queue_as :phone_calls
+
+  require 'net/http'
+  require 'open-uri'
```

Comment:
You only need `open-uri` if you use `Kernel#open` with URLs, so I think this require can be removed.

Reply:
Really? Ok, I'll remove it and test it :)